### PR TITLE
Add PermissionScheme Service

### PR DESCRIPTION
**Pull request by Weiss91**
Expand the PermissionScheme struct of ProjectService.
Add new PermissionSchemeService with functions Get and GetList.
Add corresponding mocks and tests.

### CODE REVIEW

File: permissionschemes_test.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
+package jira
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"testing"
+)
+
+func TestPermissionSchemeService_GetList(t *testing.T) {
```
Comment (ghostsquad) [inline permissionschemes_test.go]: Please add negative tests (those that validate when an error should be returned)

File: permissionschemes_test.go [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+	permissionScheme, _, err := testClient.PermissionScheme.GetList()
+	if permissionScheme == nil {
+		t.Error("Expected role list. Role list is nil")
+	}
+	if err != nil {
+		t.Errorf("Error given: %v", err)
+	}
+}
+
+func TestPermissionSchemeService_Get(t *testing.T) {
```
Comment (ghostsquad) [inline permissionschemes_test.go]: Please add negative tests (those that validate when an error should be returned

**Comment by ghostsquad** [comment]
@Weiss91 I know this is probably a small concern, and let me know if it's not a concern to you before I merge:

Could you rebase/squash you commits while following this standard for your commit message?

chris.beams.io/posts/git-commit

This will allow me to rebase & merge (instead of squash and merge) so that you get full credit for these commits.

**Comment by Weiss91** [comment]
@ghostsquad I have rebased/squashed it. 

I hope it will fit all requirements now. If there is still something missing let me know.

**Comment by ghostsquad** [comment]
I had to rebase/merge due to a conflict from a merge from another PR.

**Pull request merged**